README.md

```diff
@@ -5,7 +5,7 @@
 [![License](https://poser.pugx.org/moodlehq/moodle-plugin-ci/license)](//packagist.org/packages/moodlehq/moodle-plugin-ci)
 
 The goal of this project is to facilitate the running of tests and code
-analysis against a Moodle plugin in CI tools, such as [Travis
+analysis against a Moodle plugin using CI environment, such as [Travis
```

Comment:
"a CI environment" or "CI environments", but current is strange, isn't it?

Reply:
Fixed